corehq/apps/app_manager/templates/app_manager/v2/apps_base.html

```diff
@@ -83,8 +83,10 @@
     <script src="{% static 'hqwebapp/js/lib/jquery.textchange.min.js' %}"></script>
 
     <script src="{% static 'hqwebapp/js/stay-on-tab.js' %}"></script>
+    {% if not is_appmanager_admin %}
```

Comment:
How about moving these two scripts to `managed_app.html` instead of using this flag?